modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -44,7 +49,7 @@ $ echo -ne "root" | base64
 $ echo -ne "password" | base64
 ----
 
-. Create a configuration file for the bare metal node.
+. Create a configuration file for the bare metal node using the following example `bmh.yaml` file, replacing values in the YAML to match your environment:
```

Comment:
The network configuration secret is optional, to be used when trying to create a bmh with day1 network configuration (eg. ip address)